From: Arnaud Ferraris <aferraris@debian.org>
Date: Mon, 18 Nov 2024 15:05:19 +0100
Subject: metainfo: add missing homepage URL

The lack of this info causes `appstreamcli` to fail validating the file,
issuing the `url-homepage-missing` warning.

Forwarded: https://gitlab.postmarketos.org/postmarketOS/mobile-config-firefox/-/merge_requests/65
---
 org.postmarketos.mobile_config_firefox.metainfo.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/org.postmarketos.mobile_config_firefox.metainfo.xml b/org.postmarketos.mobile_config_firefox.metainfo.xml
index 765e6c9..1ed81f6 100644
--- a/org.postmarketos.mobile_config_firefox.metainfo.xml
+++ b/org.postmarketos.mobile_config_firefox.metainfo.xml
@@ -4,6 +4,7 @@
 
   <name>Mobile Config for Firefox</name>
   <summary>Mobile and privacy friendly configuration for Firefox (distro-independent)</summary>
+  <url type="homepage">https://gitlab.postmarketos.org/postmarketOS/mobile-config-firefox</url>
 
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>MPL-2.0</project_license>
